### PR TITLE
Several improvements to network-ups-tools

### PR DESCRIPTION
**Pull request by dominikh**

### CODE REVIEW

**Comment by xtraeme** [comment]
:+1:

File: srcpkgs/network-ups-tools/template [hunk 1]
```diff
@@ -53,6 +54,9 @@ network-ups-tools-devel_package() {
 	short_desc+=" - development files"
 	depends="lib${sourcepkg}>=${version}_${revision}"
 	pkg_install() {
-		vmove "usr/lib/*.so"
+		vmove usr/lib/*.so
```
Comment (beefcurtains) [inline srcpkgs/network-ups-tools/template]: You sure those quotes aren't needed?

Comment (xtraeme) [inline srcpkgs/network-ups-tools/template]: Yes

**Comment by dominikh** [comment]
Merged manually.

**Pull request closed without merging**